### PR TITLE
docs: fix some typo in the deploying-to-openshift doc

### PR DESCRIPTION
**Pull request by machi1990**

### CODE REVIEW

**Review by geoand — APPROVED** [review]
Thanks for spotting these!

File: docs/src/main/asciidoc/deploying-to-openshift.adoc [hunk 1]
```diff
@@ -42,7 +42,7 @@ Quarkus offers the ability to automatically generate OpenShift resources based o
 The OpenShift extension is actually a wrapper extension that brings together the link:deploying-to-kubernetes[kubernetes] and link:container-image#s2i[container-image-s2i]
 extensions with sensible defaults so that it's easier for the user to get started with Quarkus on OpenShift.
 
-When we added the `openshift` extension to the command line invocation above, the following dependency was added to the `pom.xml`
+When we added the `OpenShift` extension to the command line invocation above, the following dependency was added to the `pom.xml`
```
Comment (gsmet) [inline docs/src/main/asciidoc/deploying-to-openshift.adoc]: We usually keep the lowercase extension name when in backticks.

Comment (machi1990) [reply docs/src/main/asciidoc/deploying-to-openshift.adoc]: I thought this only applies when the full artefact is in use as `quarkus-openshift`  extension.

Comment (gsmet) [reply docs/src/main/asciidoc/deploying-to-openshift.adoc]: I would either remove the backticks and have it properly cased or keep the backticks and keep it lowercase.

Comment (machi1990) [reply docs/src/main/asciidoc/deploying-to-openshift.adoc]: Great - here is the followup PR https://github.com/quarkusio/quarkus/pull/8046. Good to know this :-)

**Pull request merged**